onprc_ehr/resources/web/onprc_ehr/window/BulkSerologyScanWindow.js

```diff
@@ -169,11 +169,12 @@ Ext4.define('ONPRC_EHR.window.BulkSerologyScanWindow', {
         {
             errors.push('Missing Date');
         }
-        //var project = this.resolveProjectByName(Ext4.String.trim(row[2]), errors, rowIdx);
 
 
-        var project = this.resolveProjectByName(Ext4.String.trim(row[13]), errors, rowIdx);
-        var name = Ext4.String.trim(row[14]);
+        var project = this.resolveProjectByName(Ext4.String.trim(row[14]), errors, rowIdx);
+
+        var name = Ext4.String.trim(row[15]);
+
```

Comment:
hard coded indexes on line 174 (row[14]) & line 176(row[15]) -- Maybe a question for onprc folks if this is working as expected.